spec/requests/host_urls_are_whitelisted_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe "host urls are checked against a whitelist", type: :request do
+  scenario "a whitelisted host is accepted and and OK response is received" do
+    get root_path, headers: {"Host" => "test.local"}
```

Comment:
I like the explicit `test.local` here as it helps me understand what the test does. 👍